src/doc/reference/glossary/cilk_scope.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Cilk_scope
```

Comment:
"Cilk_scope" should be "cilk_scope" here (with a lower-case "cilk") to match the other Cilk keywords in the glossary.